Chapters/1-4-Chapter4.md

```diff
@@ -0,0 +1,232 @@
+
+"In all my years of riding trains, they've never built at least one spacious seat for people of my condition."
+
+"Maybe none of them are living with your problem," I said.
+
+"A sliver of empathy from them and an old man could sit in peace. Instead, a patient sweet maman like you has to bear my complaints."
+
+"Charming a young maman? At your age?"
+
+"I'm often told that I say too much of what I mean. Some may consider it charming, I suppose."
+
+"Well, I believe that too many of my generation are afraid to speak, so I'm glad you're around to set us straight."
+
+He chortled, "Charming an old man. At your age?"
+
+I laughed. It was loud and it was awkward, but I knew he wouldn't mind.
+
+He began again, "so what's taking you to the capital?"
+
+I froze a bit, stuttering, but said "My jundwoman works there. Just bringing her something."
+
+"Ahh. Surprising your jundwoman. I bet she'll be excited," he said.
+
+"Heh. Yea. I hope she receives it well."
+
+"A good jundwoman would be happy that you thought to brighten her day like that."
+
+"Well we haven't been too happy with each other lately, so I hope you're right."
+
+"Yes, that will happen from time to time. I still remember once, my spouses and I were fighting about a color for our daughter's first sleeping rug. We never even asked her what she wanted; we were too obsessed with winning the fight. For weeks, It was like an obsession or a fever. We didn't speak, we didn't eat together. But then one day," he snapped the air with his fingers.
+
+"... what?" I asked. "One day, what?"
+
+"The fever is gone. And insignificant."
+
+"Did you ever have a fight that didn't stop?"
+
+He paused and looked at me as if he had seen a younger version of himself in me.
+
+"...Perhaps we should stop now."
+
+"I don't mean to pry. It's just... if you have any advice, it would be greatly appreciated."
+
+"Can't you ask your parents for advice? I don't like to get into family squabbles like this."
+
+"My mother died right before I got married. My father and nather... were never really present." I said. I hoped that opening up to him would make him do the same.
+
+"Oh no. That's terrible..." He said, visibly uncomfortable. "Well you did a splendid job raising yourself. Must be those child-rearing instincts you maman have that got you through it."
+
+"They aren't instincts. We learn them just like any other skill."
+
+"Oh but you're all so good at it."
+
+"We spend a lot of time doing it. If men and women helped us, they'd be great at it too," I retorted.
+
+He sighed, "We all have a place in society."
+
+I turned forward and stopped talking before I said something I would regret. He returned his head back to the curtain, staring between stitches at the bright void of Ara. Like a switch, we returned to being strangers on the train.
+
```

Comment:
This is a very human scene, it brings the struggles of these aliens a lot closer to home.